lib/guard/ui.rb

```diff
@@ -25,7 +25,7 @@ def logger
         @logger ||= begin
                       require "lumberjack"
                       Lumberjack::Logger.new(
-                        options.fetch(:device) { $stderr },
+                        options.fetch('device') { $stderr },
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.